File: Views/src/components/out-sourced/collect-deliver/collect-deliver-create/dto/collect-deliver-dto.ts
import { MyUser } from "src/components/authentication/dto/myUser";
import { PartnerDto } from "src/components/main/partner/dto/partner-dto";
import { CompanyDto } from "src/shared/dtos/company-dto";
import { BillingFromDto } from "./billing-from-dto";
import { DestinyDto } from "./destiny-dto";


export class CollectDeliverDto {
  id: number;
  companyId: number;
  company: CompanyDto;
  userId: number;
  user: MyUser;
  transporterId: number;
  transporter: PartnerDto;
  subjectReason: string;
  contactName: string;
  start: Date;
  price: number;
  collect: boolean;
  deliver: boolean;
  other: boolean;
  billingFrom: BillingFromDto;
  taskOverView: string;
  destiny: DestinyDto;
}

File: Application/Dto/Outsourced/CollectDeliverDto.cs
using System;

namespace Application.Dto.Outsourced
{
    public class CollectDeliverDto
    {
        public int Id { get; set; }
        public int TransporterId { get; set; }
        public PartnerDto Transporter { get; set; }
         public int CompanyId { get; set; }
        public CompanyDto Company { get; set; }
        public string Subject { get; set; }
        public string OwnerResponsible { get; set; }
        public ChargeFromDto ChargeFrom { get; set; }
        public bool Collect { get; set; }
        public bool Deliver { get; set; }
        public DateTime Start { get; set; }
        public Decimal Price { get; set; }
        public string ItemsCollected { get; set; }
        public string ItemsDelivered { get; set; }
        public string Comments { get; set; }
        public string TransporterNoregisterd { get; set; }

        public string Customer { get; set; }
        public string Partner { get; set; }

        public string NoRegisterName { get; set; }
        public string NoRegisterAddress { get; set; }
    }
}